### ISSUE TITLE
Qubes issue ticket was probably erroneously deleted

### ISSUE THREAD

**Issue by adrelanos** [QubesOS/qubes-issues#8479]
Qubes issue https://github.com/QubesOS/qubes-issues/issues/8388 was deleted.

There was nothing inappropriate in this ticket as long as it was active. It as a productive ticket and I need to take another look at it. Even if spam was there, I guess you would have deleted the spam comments instead of a whole productive ticket.

If I had to guess, it's a github bug. Happened to me before. The solution was to contact github support. Then they confirmed the bug on github side and restored the ticket.

But I believe the request has to come from a github project administrator.

> https://support.github.com/request

> Choose a topic to help us route your request quickly or [open a support ticket](https://support.github.com/contact?source=subtitle&tags=rr-general-technical).

**Comment by marmarek** [comment]
This seems to happen because @bluesteal is no longer visible user (based on a vague response from github support, I guess banned). This is some BS... In some cases the content remains visible but with "Ghost user" as the author, I'll ask further why it isn't the case here.

**Comment by andrewdavidwong** [comment]
> Even if spam was there, I guess you would have deleted the spam comments instead of a whole productive ticket.

I'm fairly certain that we (i.e., members of the Qubes OS Project) do not have the ability to delete GitHub issues, at least I don't. Even if we wanted to, I don't think GitHub exposes any kind of "delete button" for us to do that. The most we could do is file a request with GitHub support to ask them to delete an issue, which we wouldn't do, except perhaps in the most extreme circumstances, and maybe not even then. (We'd more likely just report the extreme circumstances to GitHub, then let them decide how to handle it, rather than actually requesting anything be deleted.)

**Comment by marmarek** [comment]
No, we can delete issues. I'm not sure what permission is required for that, but I have it. This ability wasn't always here. Anyway, we don't really use it. I contacted github support already, and they said specifically:
> That Issue is no longer visible to a change in the account status of the Issue author. We are unable to restore the Issue, though it may be restored if the author's account status changes.

I'm not really satisfied with this response, so I'll inquire further.

**Comment by andrewdavidwong** [comment]
> No, we can delete issues. I'm not sure what permission is required for that, but I have it. This ability wasn't always here.

Oh, my mistake. Thank you for the correction. Perhaps they added the ability to delete issues at some point, or perhaps I never had the requisite permissions to see it.

> It as a productive ticket and I need to take another look at it.

By the way, since some of us have email records of the issue and comments on it, we could probably reconstruct it as a new issue. This would just be a workaround, of course, but it would at least allow you and others access to the content.

**Comment by adrelanos** [comment]
Thank you for following up on this one!

> By the way, since some of us have email records of the issue and comments on it, we could probably reconstruct it as a new issue. This would just be a workaround, of course, but it would at least allow you and others access to the content.

That I imagine would be cumbersome. To save you some work, it's not required in this case, because the support request went into a different direction. Just wanted to see which debug output I already requested from the user and which one I already received. But not necessary now. My other findings, I already organized into Whonix wiki for later reference.

I also have e-mail records except for my own comments. Just noticed github doesn't have a setting to send myself a copy of my own issue comments because I thought that seems like a good idea to enable.

But generally, vanishing issues is a problem.

**Comment by marmarek** [comment]
> Just noticed github doesn't have a setting to send myself a copy of my own issue comments because I thought that seems like a good idea to enable.

I have it enabled, so it exists, somewhere.

**Comment by andrewdavidwong** [comment]
Profile picture in upper-right -> Settings -> Notifications -> Customize email updates -> Check "Includes your own updates"

**Comment by TNTBOMBOM** [comment]
So do we have an archive version or new ticket number/name for that one (8388)?
